### ISSUE TITLE
4.0.0-alpha.0 Icon component title attribute

### ISSUE THREAD

**Issue by tianyu94** [palantir/blueprint#4650]
#### Environment

4.0.0-alpha.0

Google Chrome

#### Question

After upgrading to 4.0, the independent icon component will set the title by default. Is there any way to remove it globally?

At the same time, the title attribute of the Button component cannot override the tittle attribute of the independent icon component 

```tsx
<Button icon={<DoubleChevronRight />} disabled={loading || !hasNext} title="Last page"
                onClick={handleLastPage} />
```

![image](https://user-images.githubusercontent.com/37231629/115100485-9aaac080-9f6f-11eb-857b-27a79fef02e3.png)

**Comment by adidahiya** [comment]
@tianyu94 thanks for pointing out the title tooltip regression. the second point you raise is a separate issue; we never had support for that in Blueprint. we might consider adding something like `iconProps` to fulfill that use case, but is it something you could accomplish with other Blueprint APIs instead? For example, why not put a Tooltip around the Button to hint at what it does?

**Comment by adidahiya** [comment]
The title regression is fixed by #4672
